WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsListFragment.java

```diff
@@ -362,6 +363,23 @@ public void onStart() {
         EventBus.getDefault().registerSticky(this);
     }
 
+    // Removes app notifications from the system bar
+    class CancelNotificationsTask extends AsyncTask<Void, Void, Void> {
```

Comment:
> charging clasp is broken and the device is almost out of power

Mine broke too! Bad form, Samsung. I was able to strap it on with some tape to get it charging again :)